resources/assets/less/bem/wiki-search.less

```diff
@@ -20,35 +20,36 @@
   padding-top: 10px;
 
   &__bar {
-    height: 54px;
-    width: 100%;
-    padding: 0 20px;
-    margin-right: -38px;
+    .default-border-radius();
+    display: flex;
+    flex-direction: row;
 
-    background-color: @osu-colour-b6;
-    color: @osu-colour-c1;
+    width: 100%;
+    padding: 13px 20px;
 
     font-size: @font-size--wiki-search;
 
+    background-color: @osu-colour-b6;
+
     border: 2px solid @osu-colour-orange-1;
-    .default-border-radius;
+  }
+
+  &__input {
```

Comment:
should also `.reset-input()` this, otherwise, Chrome and Safari do this:

![image](https://user-images.githubusercontent.com/1694544/65569262-e49c1200-df97-11e9-9fea-a332aba82600.png)